consensus/src/chained_bft/event_processor.rs

```diff
@@ -134,31 +138,42 @@ impl<T: Payload> EventProcessor<T> {
                 counters::TIMEOUT_ROUNDS_COUNT.inc();
             }
         };
+        // are we the current round's leader?
         if self
             .proposer_election
             .is_valid_proposer(self.author, new_round_event.round)
             .is_none()
         {
             return;
         }
+        // generate the proposal
```

Comment:
those comments look redundant, can we remove them? the code is pretty self-explanatory

Reply:
I didn't comment on all of them, but I'm referring to all those just repeat the next line's function name

Reply:
I don't really agree with this. I think it's good to be able to understand what a function does from a cursory glance, especially critical code. But in the interest of time I removed them : o